index.rst

```diff
@@ -45,6 +45,7 @@ NOTE: There are multiple options for reading this documentation. See link to the
 
    pages/enterprise/intro
    pages/enterprise/setup
+   pages/enterprise/cloud_forwarder
```

Comment:
This needs to point to the page itself not the parent directory of the page:

```suggestion
   pages/enterprise/cloud_forwarder/cloud_forwarder
```

The URL path for the forwarder docs will look like `/pages/enterprise/cloud_forwarder/cloud_forwarder.html`. If we want to get rid of the double "cloud_forwarder" and use `pages/enterprise/cloud_forwarder.html`, we can move the `cloud_forwarder.rst` file directly into the `pages/enterprise` folder. If we do that we also don't need the toc tree fix above.